docs/docs/performance.md

```diff
@@ -9,33 +9,41 @@ This means that performance inside an enclave will normally be lower than with a
 the performance difference and how they vary between variety of benchmarks taken from the 
 [Computer Language Benchmarks Game](https://salsa.debian.org/benchmarksgame-team/benchmarksgame/).
 
+The configuration used to perform the benchmarks were:
+1) Warmup iteration: 10 
+2) Number of iteration for each benchmark: 100
+3) Azure machine series: Standard DC4s v3 (4 vcpus - Intel(R) Xeon(R) Platinum 8370C CPU @ 2.80GHz, 32 GiB memory)
+4) Operating system: Ubuntu 20.04.5 LTS 
+5) Conclave version: 1.3
+
 !!! note
 	The "empty" benchmark is measuring the overhead of entering and exiting the enclave, without doing any
 	work. As entering/exiting triggers a variety of hardware mechanisms designed to block side channel attacks this is
 	naturally expensive relative to the cost of a regular function call, however, once the enclave is doing real work
 	this transition cost becomes less relevant.
 
-| Benchmark     | Environment|        Score |          Error |  Units |
-|---------------|------------|--------------|----------------|--------|
-| empty         |   Enclave  |    51921.076 | ±     1697.024 |  ops/s |
-| empty         |   Host     | 49453365.793 | ±  3404118.758 |  ops/s |
-| binary_trees  |   Enclave  |      454.061 | ±       31.089 |  ops/s |
-| binary_trees  |   Host     |     1758.980 | ±       79.428 |  ops/s |
-| fannkuch      |   Enclave  |        4.181 | ±        0.024 |  ops/s |
-| fannkuch      |   Host     |        5.925 | ±        0.063 |  ops/s |
-| fasta         |   Enclave  |        3.185 | ±        0.028 |  ops/s |
-| fasta         |   Host     |        4.022 | ±        0.127 |  ops/s |
-| himeno        |   Enclave  |        0.179 | ±        0.004 |  ops/s |
-| himeno        |   Host     |        0.366 | ±        0.003 |  ops/s |
-| mandelbrot    |   Enclave  |        5.529 | ±        0.134 |  ops/s |
-| mandelbrot    |   Host     |        6.385 | ±        0.132 |  ops/s |
-| nbody         |   Enclave  |        1.205 | ±        0.021 |  ops/s |
-| nbody         |   Host     |        1.279 | ±        0.017 |  ops/s |
-| pidigits      |   Enclave  |        9.941 | ±        0.185 |  ops/s |
-| pidigits      |   Host     |       24.722 | ±        0.301 |  ops/s |
-| spectral_norm |   Enclave  |       11.923 | ±        0.274 |  ops/s |
-| spectral_norm |   Host     |       17.345 | ±        0.930 |  ops/s |
+| Benchmark    | SGX   | Score (ops/s) | Score Delta (%) | Error (ops/s)  | Error (%) |
+|--------------|-------|---------------|-----------------|----------------|-----------|
+| empty        | 	Off	| 24892554.58	| N.A.            | ±7868.489      | ±0.03     |
```

Comment:
The values seem to be copied from the previous PR. Just to be sure, were these re-run on 1.3?

Reply:
Not yet. It takes some time to run the tests.

Reply:
The values are now up to date.